iot/dotnet-iot-and-nanoframework/README.md

```diff
@@ -0,0 +1,67 @@
+--- 
+languages:
+- csharp
+products:
+- dotnet
+- dotnet-core
```

Comment:
maybe add dotnet nanoFramework here as well?

Reply:
This is a pre-defined taxonomies product list, I checked nanoFramework but it's not in pre-defined list.